common/install_cuda.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+set -ex
+
+# install CUDA 9.2 in the same container
+wget -q https://developer.nvidia.com/compute/cuda/9.2/Prod2/local_installers/cuda_9.2.148_396.37_linux -O setup
+chmod +x setup
+./setup --silent --no-opengl-libs --toolkit
+rm -f setup
```

Comment:
In all of the scripts, I split `command a && command b` into `command a \n command b`, so that it's easier to see which line fails. The original script uses `&&` to concatenate multiple lines into one `RUN` command, likely to prevent Docker from creating too many layers (since it creates one layer per `RUN` command). It is not a concern now because we only run one `RUN` command per script.